examples/example-node-telemetry/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@realm/example-node-telemetry",
+  "private": true,
+  "version": "0.1.0",
+  "description": "A demonstration of how to use data ingest to store and visualize sensor data",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "lint": "eslint --ext .js,.mjs,.ts .",
```

We could either add a `start` script that includes compiling (below), or just have it be `node dist/app.js` and expect the `build` step to be run prior 👍 
```suggestion
    "lint": "eslint --ext .js,.mjs,.ts .",
    "start": "npm run build && node dist/app.js",
```